x-pack/plugins/ml/public/util/ml_time_buckets.js

```diff
@@ -15,16 +15,16 @@ import _ from 'lodash';
 import moment from 'moment';
 import dateMath from '@elastic/datemath';
 
-import { TimeBucketsCalcAutoIntervalProvider } from 'plugins/ml/util/ml_calc_auto_interval';
+import { timeBucketsCalcAutoIntervalProvider } from 'plugins/ml/util/ml_calc_auto_interval';
 import { inherits } from 'plugins/ml/util/inherits';
 
 const unitsDesc = dateMath.unitsDesc;
 const largeMax = unitsDesc.indexOf('w');    // Multiple units of week or longer converted to days for ES intervals.
 
 import { TimeBuckets } from 'ui/time_buckets';
-export function IntervalHelperProvider(Private, config) {
+export function IntervalHelperProvider(config) {
```

Comment:
Config could be removed as a parameter and brought in from client.
`const config = chrome.getUiSettingsClient();`
This would then match the changes which currently exist on a feature branch.
https://github.com/jgowdyelastic/kibana/blob/49fbc41ccd9474bf6040b7a038c69b8a2cdf882d/x-pack/plugins/ml/public/util/ml_time_buckets.js#L26